src/libraries/System.Private.CoreLib/src/System/Runtime/Loader/AssemblyLoadContext.cs

```diff
@@ -12,8 +12,10 @@
 
 namespace System.Runtime.Loader
 {
+    [StructLayout(LayoutKind.Sequential)]
```

Comment:
Unfortunately I specifically need the fields in the shared portion. I think it's necessary, but I'll confirm and then we can figure out how to proceed. cc: @vargaz

Reply:
The `Sequential` attribute applies to the whole type. I do not think it matters whether it is in the Mono-specific or shared portions.

Reply:
Oh, you mean put the attribute in the Mono partial class file so that it's only applied when SPC is built for Mono? I think I misunderstood what you were asking, but I can definitely do that.

Reply:
> you mean put the attribute in the Mono partial class file 

Yep